### PR TITLE
Debugging issues with archiving and restoring WCS solutions from headerlets

### PR DESCRIPTION
**Pull request by mdlpstsci**
Code currently appears to be working fine now with traps for exceptions.  I just need to understand why some of the exceptions are being raised in the first place.  Examining the headerlet code in stwcs.

### CODE REVIEW

File: drizzlepac/hapsequencer.py [hunk 1]
```diff
@@ -74,6 +74,8 @@
 from stwcs import wcsutil
 from stwcs.wcsutil import headerlet
 
+import pdb
```
Comment (stsci-hack) [inline drizzlepac/hapsequencer.py:77]: This should not remain in operational code.

File: drizzlepac/hapsequencer.py [hunk 2]
```diff
@@ -1035,6 +1037,8 @@ def update_wcs_in_visit(tdp):
         log.info("The active WCS solution for each Grism/Prism and direct image is not changed.")
 
     log.info("Grism product list: {}".format(grism_product_list))
+    # MDD
+    sys.exit(0)
```
Comment (stsci-hack) [inline drizzlepac/hapsequencer.py:1041]: Did you mean to exit before returning the list?

File: drizzlepac/hapsequencer.py (hunk 1 above)
Comment (mdlpstsci) [reply drizzlepac/hapsequencer.py]: Sorry, code was still "In progress".  This line of code is deleted.

File: drizzlepac/hapsequencer.py (hunk 2 above)
Comment (mdlpstsci) [reply drizzlepac/hapsequencer.py]: Sorry, code was still "In Progress" and being debugged.  The lines 1040/1041 have been removed.

**Review by stsci-hack — APPROVED** [review]
LGTM

**Pull request merged**